docs/using-jda/interactions.md

```diff
@@ -96,64 +97,116 @@ When you use `deferReply` the first message sent to this webhook will act identi
+
+### Handling ButtonClickEvent
+
+When a user presses one of these buttons, you will receive a `ButtonClickEvent` for the respective interaction.
+
+Each non-link button requires such an ID in order to be used.
 
 !!! example
 
-    ```java
-    public class HelloBot extends ListenerAdapter {
-      @Override
-      public void onSlashCommand(SlashCommandEvent event) {
-          if (event.getName().equals("hello")) {
-              event.reply("Click the button to say hello")
-                  .addActionRow(
-                    Button.primary("hello", "Click Me"), // Button with only a label
-                    Button.success("emoji", Emoji.fromMarkdown("<:minn:245267426227388416>"))) // Button with only an emoji
-                  .queue();
-          } else if (event.getName().equals("info")) {
-              event.reply("Click the buttons for more info")
-                  .addActionRow( // link buttons don't send events, they just open a link in the browser when clicked
-                      Button.link("https://github.com/DV8FromTheWorld/JDA", "GitHub")
-                        .withEmoji(Emoji.fromMarkdown("<:github:849286315580719104>")), // Link Button with label and emoji
-                      Button.link("https://ci.dv8tion.net/job/JDA/javadoc/", "Javadocs")) // Link Button with only a label
-                  .queue();
-          }
-      }
+```java
+ public class HelloBot extends ListenerAdapter {
+   @Override
+   public void onSlashCommand(SlashCommandEvent event) {
+       if (event.getName().equals("hello")) {
+           event.reply("Click the button to say hello")
+               .addActionRow(
+                 Button.primary("hello", "Click Me"), // Button with only a label
+                 Button.success("emoji", Emoji.fromMarkdown("<:minn:245267426227388416>"))) // Button with only an emoji
+               .queue();
+       } else if (event.getName().equals("info")) {
+           event.reply("Click the buttons for more info")
+               .addActionRow( // link buttons don't send events, they just open a link in the browser when clicked
+                   Button.link("https://github.com/DV8FromTheWorld/JDA", "GitHub")
+                     .withEmoji(Emoji.fromMarkdown("<:github:849286315580719104>")), // Link Button with label and emoji
+                   Button.link("https://ci.dv8tion.net/job/JDA/javadoc/", "Javadocs")) // Link Button with only a label
+               .queue();
+       }
+   }
+
+   @Override
+   public void onButtonClick(ButtonClickEvent event) {
+       if (event.getComponentId().equals("hello")) {
+           event.reply("Hello :)").queue(); // send a message in the channel
+       } else if (event.getComponentId().equals("emoji")) {
+           event.editMessage("That button didn't say click me").queue(); // update the message
+       }
+   }
+ }
+```
```

Comment:
This section has to be indented by 4 spaces, like it originally was, for it to show as part of the admonition.